packages/cli/src/chat/chat-service.ts

```diff
@@ -266,7 +266,7 @@ export class ChatService {
 	private stringifyRawData(data: RawData) {
 		const buffer = Array.isArray(data)
 			? Buffer.concat(data.map((chunk) => Buffer.from(chunk)))
-			: Buffer.from(data);
+			: Buffer.from(data instanceof ArrayBuffer ? new Uint8Array(data) : data);
```

Comment:
`ArrayBuffer` is no longer a supertype of `TypedArray` types like `Buffer` and `Uint8Array`.